Remotion/ObjectBinding/Web.Development.WebTesting/ControlObjects/BocAutoCompleteReferenceValueControlObject.cs

```diff
@@ -373,11 +376,7 @@ private IReadOnlyList<SearchServiceResultItem> ParseSearchServiceResponse ([NotN
       switch (state)
       {
         case AutoCompleteSearchService.Success:
-          var successData = (IReadOnlyCollection<object>)data;
-          return successData.Cast<IDictionary<string, object>>()
-              .Where(d => d != null) // empty JSON object (= no result) is converted to null
-              .Select(d => new SearchServiceResultItem((string)d["UniqueIdentifier"], (string)d["DisplayName"], (string)d["IconUrl"]))
-              .ToList();
+          return successParser((IReadOnlyCollection<object>)data);
```

Comment:
Wouldn't this always be successParser(((IReadOnlyCollection<IDictionary<string, object>>)data).Single()) for the forseeable future? 

Is there an expected/known error scenario where the result is 200 but empty so .Single() would fail or can we expect this to never actually happen. Reasoning: Single() is a very unhelpful API in case the expectation is violated. You can (hopefully) figure things out with the stacktrace but a meaningful msg is usually preferred.

Reply:
Gut feeling says it would be a good idea. (especially since have some other details anyway in this PR).